### DISCUSSION TITLE
这个动态展开功能为什么就我有问题呢？之前还以为是出bug了

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [the1812/Bilibili-Evolved#4650] [提问 / Q&A]
https://github.com/the1812/Bilibili-Evolved/issues/4633

被上了个无法复现的标志，可是我两个浏览器都不行，其他插件也都好好的
假如没办法的话那有没有别的方法可以实现一样的功能呢？用习惯了现在每次都要点展开好烦啊

**Comment by Tinhone** [comment]
我这也没问题，要不把控制台全部的日志发上来看看？或许可以知道发生了什么

**Reply by KuiPer-LTW** [reply]
具体要怎么发，我小白不懂？之前发issues的时候也是没看到示例里的异常才没发的

**Reply by the1812** [reply]
去 #4633 看看, 复制出 outerHTML
